ConwaysGameOfLife/Generator.cs

```diff
@@ -36,35 +36,37 @@ public Universe GenerateNewUniverse()
         {
             var oldCells = _oldUniverse.Cells;
             var newCells = new List<Cell>();

```

Comment:
this can be moved to GenerateCells as you are only needing the empty list in there

```suggestion
            var newCells = GenerateCells(oldCells);

```